SA/design_concept/content/tool_research/enso.typ

```diff
@@ -0,0 +1,16 @@
+= Enso
+Enso is a functional programming language created by Enso International Inc., designed for data science.
+Programs can be created by both text and a visual editor.
+The visual editor allows a user to define components that can be connected with each other, symbolizing the data-flow from one component to another.
+The editor also offers previews for a component's data, e.g. allowing a user to see a modified picture like in @enso_screenshot.
+@enso-language
```

Comment:
```suggestion
```

Reply:
Now I see, that reference is meant for the complete paragraph.

Maybe put it a bit higher and reference it in the text?

Reply:
I'd like to leave this in, to make it clear that that paragraphy is based on information from enso, and the text paragraph on my impressions.